src/compressor.h

```diff
@@ -19,6 +19,7 @@ bool CompressScript(const CScript& script, std::vector<unsigned char> &out);
 unsigned int GetSpecialScriptSize(unsigned int nSize);
 bool DecompressScript(CScript& script, unsigned int nSize, const std::vector<unsigned char> &out);
 
+// Defined for nAmount <= MAX_MONEY.
```

Comment:
Do you have any evidence to support this claim?

Reply:
The algorithm works for larger numbers fine. Maybe just reword the comment a bit?

https://ideone.com/6TkUTp for tinkering

Reply:
I believe the algorithm works fine up to approximately 2^64 / 9.

Reply:
Yes, it is the hard upper limit at `2^64 / 9` that I noticed: above that we wrap around :)

Reply:
@kiminuo The point of the comment is to make it clear that this function is not intended to be used with input values above `MAX_MONEY` :)

Reply:
@practicalswift I know you know. Not sure if others know. :) (Ideone snippet was for me to understand the algorithm better.)